reference-architectures/cloud_deploy_flow/main.tf

```
@@ -0,0 +1,176 @@
+# Configure the Google Cloud provider
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 4.0"
```

Let's unbind this from v4 unless you have a specific reason to (we are at v6.11 now)
```suggestion
      version = "> 4.0"
```

---

Done